### PR TITLE
fix error when hide exposure flag is false

### PR DESCRIPTION
**Pull request by myarjunar**
### What does it fix?
* Ticket: #
* Funded by: DFAT
* Description: fix error when hide exposure flag is false

### Checklist:
- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

**Comment by Gustry** [comment]
Hum, it might be a good catch
I totally forgot this option, and I wonder which behaviour we should have when we set the custom order. I think we must discard it

Is-it fixing the issue we mentioned with Tim?

**Comment by myarjunar** [comment]
shout out to your french exception message :grinning: I don't know about Tim, he may have a different error.

> I think we must discard it

I just apply the same behaviour as single IF, lets wait until they say otherwise :smiley:

**Comment by Gustry** [comment]
Can you use definitions?

File: safe/report/extractors/composer.py [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 from safe.common.version import get_version
 from safe.definitions.fields import analysis_name_field
+from safe.definitions.provenance import provenance_exposure_layer_id, \
```
Comment (ismailsunni) [inline safe/report/extractors/composer.py]: backslash

**Pull request merged**